### ISSUE TITLE
[0.10.0] Continuous query only execute on one database when name is identical on two databases

### ISSUE THREAD

**Issue by jeanpralo** [influxdata/influxdb#5812]
Hi there,

I have two databases, db1 and db2. On those databases I have the same retention policies and the same continuous queries.

I realized that only the CQ from db1 was being executed and the ones from db2 were never launched, I then tried to change the name of the CQ on db2 and they started to be run properly.

To reproduce:

```
CREATE DATABASE db1
CREATE DATABASE db2

REATE RETENTION POLICY "_1h" on db1 DURATION 1h REPLICATION 1
REATE RETENTION POLICY "_1h" on db2 DURATION 1h REPLICATION 1


CREATE CONTINUOUS QUERY "_1h" ON db1 BEGIN SELECT mean(value) as value INTO "db1"."_1h".:MEASUREMENT FROM "db1"."default"./.*/ GROUP BY time(1m),host END
CREATE CONTINUOUS QUERY "_1h" ON db2 BEGIN SELECT mean(value) as value INTO "db2"."_1h".:MEASUREMENT FROM "db2"."default"./.*/ GROUP BY time(1m),host END
````

Insert data into the default RP for db1 and db2, and check the logs, you will only see this:
```
[continuous_querier] 2016/02/23 20:41:00 executing continuous query _1h (2016-02-23 20:40:00 +0000 UTC to 2016-02-23 20:41:00 +0000 UTC)
[query] 2016/02/23 20:41:00 SELECT mean(value) AS value INTO db1._1h.:MEASUREMENT FROM db1."default"./.*/ WHERE time >= '2016-02-23T20:40:00Z' AND time < '2016-02-23T20:41:00Z' GROUP BY time(1m), host

```

In order to fix it just create a CQ on db2 with a different name:

CREATE CONTINUOUS QUERY "_1h2" ON db2 BEGIN SELECT mean(value) as value INTO "db2"."_1h".:MEASUREMENT FROM "db2"."default"./.*/ GROUP BY time(1m),host END

And check your logs again:
```
[continuous_querier] 2016/02/23 20:43:00 executing continuous query _1h (2016-02-23 20:42:00 +0000 UTC to 2016-02-23 20:43:00 +0000 UTC)
[query] 2016/02/23 20:43:00 SELECT mean(value) AS value INTO db1._1h.:MEASUREMENT FROM db1."default"./.*/ WHERE time >= '2016-02-23T20:42:00Z' AND time < '2016-02-23T20:43:00Z' GROUP BY time(1m), host

--
[continuous_querier] 2016/02/23 20:43:15 executing continuous query _1h2 (2016-02-23 20:42:00 +0000 UTC to 2016-02-23 20:43:00 +0000 UTC)
[query] 2016/02/23 20:43:15 SELECT mean(value) AS value INTO db2._1h.:MEASUREMENT FROM db2."default"./.*/ WHERE time >= '2016-02-23T20:42:00Z' AND time < '2016-02-23T20:43:00Z' GROUP BY time(1m), host
```

**Comment by exeral** [comment]
I confirm, I had the same issue in the past (0.9.x)
and did the same fix (different names)

This should be specified in documentation

**Comment by jsternberg** [comment]
Closing this in favor of #5814.

@rkuchan since this isn't a 0.10 specific bug, can we include a warning in the docs about this with a note that it will be fixed in 0.11?
